backend/gce/api.go

```diff
@@ -94,13 +110,20 @@ func newAPI() (*gceAPI, error) {
 		return nil, fmt.Errorf("error getting instance from compute service: %v", err)
 	}
 
+	if len(gi.NetworkInterfaces) != 1 {
```

Comment:
I guess I'm not sure how we would determine which network interface gets the alias IP. Are you thinking we could just choose the first interface listed and ignore the others?

Reply:
Changed this check to return an error when there are zero network interfaces to avoid crashing during future codepaths. Should tolerate multiple network interfaces now.